test/e2e/generate_kube_test.go

```diff
@@ -1562,4 +1562,53 @@ USER test1`
 		Expect(pod.Name).To(Equal("testpod"))
 		Expect(pod.Spec.Hostname).To(Equal(""))
 	})
+
+	It("podman generate kube --no-trunc on container", func() {
+		dirName := "demoDir"
+		ctrName := "demo"
+		err := os.Mkdir(dirName, os.ModePerm)
```

Comment:
Please follow existing convention of `Mkdir`()ing within `tempdir`.

Reply:
Thanks for the feedback, Ed. I'll work on putting something better together